content/source/docs/enterprise/api/policies.html.md

```diff
@@ -27,7 +27,9 @@ sidebar_current: "docs-enterprise2-api-policies"
 
 Policies are configured on a per-organization level and are organized and grouped into [policy sets](../sentinel/manage-policies.html#organizing-policies-with-policy-sets), which define the workspaces on which policies are enforced during runs. In these workspaces, the plan's changes are validated against the relevant policies after the plan step. (For details, see [Run States and Stages](../run/states.html).)
 
-This page documents the API endpoints to create, read, update, and delete the Sentinel policies in an organization. To view and manage the results of a specific run's policy check, use the [Runs API](./run.html).
+This page documents the API endpoints to create, read, update, and delete the Sentinel policies in an organization. Use of these endpoints provides a method to manually manage individual policies within Terraform Enterprise. To view and manage the results of a specific run's policy check, use the [Runs API](./run.html).
+
+~> **Note**: Use of these endpoints is not required to use Sentinel features in Terraform Enterprise. Alternatively, you can manage versioned policy sets, which allows integrating with VCS or uploading policy set data and configuration as a whole. Use the [policy sets API](./policy-sets.html) to leverage versioned policy sets.
```

Comment:
I wonder if this should be more strongly worded — "These endpoints are no longer the recommended way to use Sentinel features in Terraform Enterprise. We suggest using..."